src/status_im/ui/screens/wallet/history/views.cljs

```diff
@@ -47,25 +47,29 @@
    [btn/primary-button {:text (i18n/label :t/transactions-sign) :on-press #(on-sign-transaction m)}]
    [btn/secondary-button {:text (i18n/label :t/delete) :on-press #(on-delete-transaction m)}]])
 
-(defn- unsigned? [state] (= "unsigned" state))
+(defn- unsigned? [type] (= "unsigned" type))
+(defn- inbound? [type] (= "inbound" type))
 
 (defn- transaction-icon [k color] {:icon k :style (history-styles/transaction-icon-background color)})
 
 (defn- transaction-type->icon [s]
   (case s
-    "incoming" (transaction-icon :icons/arrow-left :red)
-    "outgoing" (transaction-icon :icons/arrow-right :red)
+    "inbound" (transaction-icon :icons/arrow-left :red)
```

Comment:
maybe let's move it to constant? We are using "inbound" twice, so now we have some chances to make a typo. The same for "outbound"

Reply:
I don't think it is necessary to make it a constant because it is really only used here, elsewhere it is a keyword but it is turned into a string here by the component afaik

Reply:
be "here" do you mean this ns? It's used at line 51